src/config/networkConfig.ts

```diff
@@ -365,6 +365,23 @@ export class NetworkConfig {
     }
   }
 
+  getOracleFromName(oracleName: KnownOracle): Oracle {
+    const oracles = networks[this.networkId].oracles
+
+    for (const oracle of oracles) {
+      if (oracle.name === oracleName) {
+        return oracle
+      }
+    }
+
+    return {
+      name: 'unknown',
+      description: 'Unknown',
+      url: '',
+      address: '',
+    }
+  }
+
```

Comment:
If unknown oracle is named `unknownOracle` thsi could also be written like
`networks[this.networkId].oracles.find(o => oracle.name === oracleName) || unknownOracle`, to me is shorter and clearer but is a matter of taste, so take it with a grain of salt

Reply:
Mmm I will not do this, as you say is a matter of taste